### PR TITLE
Add Stackdriver Logging suites

### PR DESCRIPTION
**Pull request by crassirostris**
Verbose Stackdriver Logging e2e testing requires separate suites, mainly for the lower logging verbosity, as it may influence the results.

Additionally, this PR lowers the logging verbosity for Elasticsearch e2e tests.

### CODE REVIEW

**Comment by piosz** [comment]
@gmarek

**Comment by crassirostris** [comment]
I tried not to modify `bootstrap-ci.yaml`, as suggested in https://github.com/kubernetes/test-infra/blob/master/README.md, but it fails the tests

**Comment by crassirostris** [comment]
@fejta verify doesn't work even on master (`bazel run testgrid/jenkins_verify:jenkins_verify -- $(pwd)/jobs/ $(pwd)/prow/ $(pwd)/testgrid/config/config.yaml`)

**Comment by fejta** [comment]
@spxtr @rmmh https://prow.k8s.io/log?pod=pull-test-infra-verify-bazel-441

**Comment by fejta** [comment]
Looks like one problem here is the verify big dies not define an upload path. 

The other problem is that we need to update bazel files.

**Comment by crassirostris** [comment]
@fejta Thanks for digging this so early in the morning! :)

**Comment by fejta** [comment]
@k8s-bot verify test this

**Comment by fejta** [comment]
@k8s-bot verify test this

**Comment by crassirostris** [comment]
@fejta Thanks!

File: jobs/ci-kubernetes-e2e-gce-sd-logging.env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### job-env
+PROJECT=k8s-jkns-gce-sd-log
```
Comment (fejta) [inline jobs/ci-kubernetes-e2e-gce-sd-logging.env]: There's an internal script we use to ensure this file has the right grants. Did you use this?

**Comment by fejta** [comment]
Structurally this looks good. Will let Marek and/or someone else ensure the environment configs do what you want them to do.

File: jobs/ci-kubernetes-e2e-gce-sd-logging.env (hunk 1 above)
Comment (crassirostris) [inline jobs/ci-kubernetes-e2e-gce-sd-logging.env]: You mean [check_projects.py](https://github.com/kubernetes/test-infra/blob/master/jenkins/check_projects.py)? If so, yes, I did

Comment (crassirostris) [inline jobs/ci-kubernetes-e2e-gce-sd-logging.env]: It reported some problem, but not on the projects I added

Comment (fejta) [inline jobs/ci-kubernetes-e2e-gce-sd-logging.env]: Great thanks!

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml [hunk 2]
```diff
@@ -683,6 +683,12 @@
         timeout: 90
         frequency: 'H/5 * * * *' # At least every 30m
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-sd-logging:
+        job-name: ci-kubernetes-e2e-gce-sd-logging
+        jenkins-timeout: 220
+        timeout: 120
```
Comment (piosz) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: Is this enough? What is the max duration of our tests?

Comment (crassirostris) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: Max theoretical duration is ~1h20m

**Pull request merged**